.editorconfig

```diff
@@ -12,6 +12,7 @@ trim_trailing_whitespace = true
 # Code files
 [*.{cs,csx,vb,vbx}]
 indent_size = 4
+insert_final_newline = true
```

Comment:
I'm personally fine with this but thought we'd purposefully left the final newline up to individual devs' preference❔

Reply:
In this repo specifically the (sub-)team is in the habit of pointing out when there's a missing newline at the end as a PR comment. Given that I figured it was better to put that notification on the inner loop.